client/app/components/packages/pas-form/edit.hbs

```diff
@@ -46,56 +46,47 @@
       />
-
     </div>
+
     <div class="cell large-4">
       <pasForm.StickySidebar>
         <pasForm.SaveButton
           @isEnabled={{or @package.isDirty pasForm.isSaveable}}
           @onClick={{this.savePackage}}
           data-test-save-button
         />
-        {{!-- Error Handling --}}
+
         <Packages::PasForm::PasFormError @package={{@package}} />
         <pasForm.SubmitButton
-          @onClick={{this.toggleModal}}
           @isEnabled={{pasForm.isSubmittable}}
           class="secondary"
           data-test-submit-button
         />
       </pasForm.StickySidebar>
-      {{#if this.modalIsOpen}}
-        <EmberWormhole @to="reveal-modal-container">
-          <RevealModal @toggle={{this.toggleModal}}>
-            <h6>Confirm Pre-Applicant Statement submission</h6>
-            <p class="header-large medium-margin-top small-margin-bottom">Are you sure?</p>
-            <p>Before submitting, ensure that your answers are accurate and complete, and that necessary attachments (including the signature form) have been uploaded. If City Planning does not receive enough accurate information to provide guidance, the lead planner will notify you and request that this form be resubmitted with needed materials, corrections, or clarifications.</p>
-            <hr>
-            <div class="grid-x">
-              <div class="cell large-6 small-padding-right">
-                <button
-                  type="button"
-                  class="button expanded large secondary"
-                  disabled={{pasForm.saveChangeset.isRunning}}
-                  {{on "click" this.toggleModal}}
-                >
-                  Continue Editing
-                </button>
-              </div>
-              <div class="cell large-6 small-padding-left">
-                <pasForm.SubmitButton
-                  @onClick={{this.submitPackage}}
-                  @isEnabled={{pasForm.isSubmittable}}
-                  class="no-margin"
-                  data-test-confirm-submit-button
-                />
-              </div>
-            </div>
-            {{!-- Error Handling --}}
-            <Packages::PasForm::PasFormError @package={{@package}}/>
-          </RevealModal>
-        </EmberWormhole>
-      {{/if}}
+
+      <pasForm.ConfirmationModal>
+        <:message>
+          <h6>Confirm Pre-Applicant Statement submission</h6>
+          <p class="header-large medium-margin-top small-margin-bottom">Are you sure?</p>
+          <p>Before submitting, ensure that your answers are accurate and complete, and that necessary attachments (including the
+            signature form) have been uploaded. If City Planning does not receive enough accurate information to provide guidance,
+            the lead planner will notify you and request that this form be resubmitted with needed materials, corrections, or
+            clarifications.</p>
+        </:message>
+
+        <:action>
+          <pasForm.SubmitButton
```

Comment:
Just another nit picky thing I'm noticing -- also maybe not relevant to this specific PR but figured I'd note it anyway. Noticing that we use `pasForm` in `pas-form/edit.hbs` here but then use `saveableForm` in `rwcds-form/edit.hbs`. Just for consistency, should probably update rwcds-form.

Reply:
@trbmcginnis yeah, I'm not sure what is the best convention. `pasForm` seems right because it makes it clear that it's validating that kind of thing and that the scope has options for that stuff.